apps/src/templates/manageStudents/ManageStudentsTable.jsx

```diff
@@ -484,6 +508,43 @@ class ManageStudentsTable extends Component {
     };
   }
 
+  familyNameColumn(sortable) {
+    return {
+      property: 'familyName',
+      header: {
+        label: i18n.familyName(),
+        props: {
+          style: {
+            ...tableLayoutStyles.headerCell,
+          },
+        },
+        transforms: [
+          (label, columnInfo) => ({
+            ...sortable(label, columnInfo),
+            children: (
+              <>
+                {React.cloneElement(sortable(label, columnInfo).children, {
+                  key: '1',
+                })}
+                <HelpTip key="2">
+                  <p>{i18n.familyNameHelpTip()}</p>
+                </HelpTip>
+              </>
+            ),
```

Comment:
this seems hacky, but it's the only way i could find to get the helptip into the header without losing the sorting buttons